autogen/oai/gemini.py

```diff
@@ -142,7 +166,7 @@ def create(self, params: Dict) -> ChatCompletion:
         elif model_name == "gemini-pro-vision":
             # B. handle the vision model
             # Gemini's vision model does not support chat history yet
-            model = genai.GenerativeModel(model_name)
+            model = genai.GenerativeModel(model_name, generation_config=generation_config, safety_settings=safety_settings)
```

Comment:
Make this Black formatter compliant (replace `\n` in params for `genai.GenerativeModel`):
line 169: `model = genai.GenerativeModel(model_name, generation_config=generation_config, safety_settings=safety_settings)` to `model = genai.GenerativeModel(\nmodel_name, generation_config=generation_config, safety_settings=safety_settings\n)`